alws/routers/sign_task.py

```diff
@@ -40,7 +47,28 @@ async def get_sign_tasks(build_id: int = None,
 async def create_sign_task(payload: sign_schema.SignTaskCreate,
                            db: database.Session = Depends(get_db),
                            user=Depends(get_current_user)):
-    return await sign_task.create_sign_task(db, payload, user.id)
+    try:
+        return await sign_task.create_sign_task(db, payload, user.id)
+    except DataNotFoundError as e:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=str(e),
+        )
+    except BuildAlreadySignedError as e:
+        raise HTTPException(
+            status_code=status.HTTP_409_CONFLICT,
+            detail=str(e),
+        )
+    except ValueError as e:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail=str(e),
+        )
+    except PermissionDenied as e:
+        return HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            detail=str(e),
+        )
```

Comment:
We can define catching exceptions through middleware like we do for products (see `alws/middlewares/products.py`)